Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAli_cff.py

```diff
@@ -0,0 +1,164 @@
+import FWCore.ParameterSet.Config as cms
+
+# ------------------------------------------------------------------------------
+# configure a filter to run only on the events selected by TkAlMinBias AlcaReco
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+ALCARECOTkAlMinBiasFilterForSiPixelAli = copy.deepcopy(hltHighLevel)
+ALCARECOTkAlMinBiasFilterForSiPixelAli.HLTPaths = ['pathALCARECOTkAlMinBias']
+ALCARECOTkAlMinBiasFilterForSiPixelAli.throw = True ## dont throw on unknown path names
+ALCARECOTkAlMinBiasFilterForSiPixelAli.TriggerResultsTag = cms.InputTag("TriggerResults","","RECO")
+
+from Alignment.CommonAlignmentProducer.LSNumberFilter_cfi import *
+
+
+# Ingredient: offlineBeamSpot
+from RecoVertex.BeamSpotProducer.BeamSpot_cfi import offlineBeamSpot
+
+# Ingredient: AlignmentTrackSelector
+# track selector for HighPurity tracks
+#-- AlignmentTrackSelector
+from Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi import AlignmentTrackSelector
+SiPixelAliHighPuritySelector = AlignmentTrackSelector.clone(
+        applyBasicCuts = True,
+        #filter = True,
+        src = 'ALCARECOTkAlMinBias',
+        trackQualities = ["highPurity"],
+        pMin = 4.,
+        )
+
+
+
+# track selection for alignment
+from Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi import AlignmentTrackSelector
+SiPixelAliTrackSelector = AlignmentTrackSelector.clone(
+	src = 'SiPixelAliTrackFitter',
+	applyBasicCuts = True,
+	pMin = 8.,
+	ptMin = 1.0,
+	etaMin = -999.,
+	etaMax = 999.,
+	nHitMin = 8,
+	nHitMin2D = 2,
+	chi2nMax = 9999.,
+	applyMultiplicityFilter = False,
+	maxMultiplicity = 1,
+	applyNHighestPt = False,
+	nHighestPt = 1,
+	seedOnlyFrom = 0,
+	applyIsolationCut = False,
+	minHitIsolation = 0.8,
+	applyChargeCheck = False,
+	minHitChargeStrip = 30.,
+)
+#Special option for PCL
+SiPixelAliTrackSelector.minHitsPerSubDet.inPIXEL = 2
+
+
+# Ingredient: SiPixelAliTrackRefitter0
+# refitting
+from RecoTracker.TrackProducer.TrackRefitters_cff import *
```

Comment:
@cerminar this inclusion here is causing the issues in the relval tests. There is duplication of definition of the `SiPixelClusters`. You might want to follow the recipe I used here https://github.com/cms-sw/cmssw/pull/9104/files to fix the SiStripAPVGain PCL